tests/phpunit/class-wc-stripe-connect-test.php

```
@@ -206,6 +206,126 @@ public function test_save_stripe_keys_decommissions_previous_webhook_before_savi
+		$method->setAccessible( true );
+		$return = $method->invoke( $this->connect, $result, 'connect', 'live' );
+
+		// The live-webhook failure is still surfaced to the caller.
+		$this->assertInstanceOf( WP_Error::class, $return );
+
+		// ...but the dual-fetched test keys were persisted regardless of it.
+		$settings = WC_Stripe_Helper::get_stripe_settings();
+		$this->assertSame( 'sk_test_123', $settings['test_secret_key'] );
+		$this->assertSame( 'pk_test_123', $settings['test_publishable_key'] );
+		$this->assertSame( 'connect', $settings['test_connection_type'] );
+	}
```

PHPMD (2.15.0)

[error] The method test_save_stripe_keys_persists_test_keys_even_when_live_webhook_fails is not named in camelCase. (undefined)

(CamelCaseMethodName)